### PR TITLE
Share

### PR DESCRIPTION
**Pull request by mikehrom**
The type of this PR is: **Instrumentation**

This PR resolves [CX-832], [AS-1889]

### Description

Creating a `share` event for use in force or eigen [[specs]](https://www.figma.com/file/oI0G7GRmNy0lBztSBczRsV/Instagram-Share?node-id=105%3A1)

### PR Checklist

- [x] If I've added a new file to the tree I've exported it from the common `index.ts`
- [x] I've added comments with examples for any new interfaces or helpers and ensured that they're in the docs 
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[CX-832]: https://artsyproduct.atlassian.net/browse/CX-832
[AS-1889]: https://artsyproduct.atlassian.net/browse/AS-1889

### CODE REVIEW

File: src/Schema/Values/Service.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * All available share services
+ *
+ * @packageDocumentation
+ */
+export enum Service {
```
Comment (mikehrom) [inline src/Schema/Values/Service.ts:6]: This list is probably not exhaustive, so we can add to it now if necessary

**Comment by pvinis** [comment]
This seems good for when we share on something. What about if the user cancels the share, or when opening the sharesheet? Do we care?

**Comment by mikehrom** [comment]
I think for now, we can implement without error states/cancel states. I don't anticipate analysis involving them -- even though I do think at some point they might be valuable to track.

For the share sheet, what I mainly want to know is:
1. how many times are users sharing artworks (or other pages)?
2. when they share, which service are they using?

If we can get all of that in one event, that's preferable. If you think pairing would make this clearer to me, let's do that in the coming week. I might be missing something about the flow itself.

**Comment by pvinis** [comment]
Closing in favor of #162.

**Pull request closed without merging**